Event.js

```diff
@@ -1,22 +1,57 @@
-//creare un input selezione tramire CreateElement("")collegandolo all'html
-// al click-->(addEventListener())parte la fetch
-//dove l'utente puo scegliere gli invitati, tramite fetch
-//cioè tutti quelli che si sono iscritti all'app
-const response = [
-  { id: 1, name: "Ani" },
-  { id: 2, name: "Eni" },
-  { id: 3, name: "Fulvio" },
-];
-
-const select = document.querySelector("#guest");
-
-for (const i of response) {
-  const options = document.createElement("option");
-  options.text = i.name;
-  select.options.add(options);
+const form_event = document.querySelector("#form_event");
+//creare una fetch per richiamare i dati dell'utente in corso
+
+// prendiamo il token identificativo per farci riconoscere dal server (il getItem vuole solo la chiave)
+const current_token = localStorage.getItem("access_token");
+
+async function getMe() {
+  const me_response = await fetch("https://iopasso.kopiro.me/me", {
```

Comment:
There's no need here to be specific about naming (`me_response`), since the function already is named `getMe()`. To simplify, simply call it `response`.

Reply:
I've changed the name in event.js and form_event.html. Also, I've translated all the code comments in en in these two files